core/src/main/resources/db/changelog/changesets/proxy_data_model.yaml

```
@@ -0,0 +1,27 @@
+databaseChangeLog:
+  - changeSet:
+      id: create_enum_relationship
+      author: pegah
+      changes:
+        - sql:
+            sql: CREATE TYPE relationship AS ENUM ('proxy', 'guardian', 'parent', 'mother', 'father', 'other');
+  - changeSet:
+      id: "proxy_data_model"
+      author: pegah
+      changes:
+        - createTable:
+            tableName: enrollee_relation
+            columns:
+              - column: { name: id, type: uuid, defaultValueComputed: gen_random_uuid(), constraints: { nullable: false, primaryKey: true } }
+              - column: { name: participant_user_id, type: uuid, constraints: {#participant user id is the user who is the proxy but not the enrollee
+                nullable: false, foreignKeyName: fk_participant_enrollee_relation_participant_user, references: participant_user(id)} }
```

for naming foreign keys, fk_[tableName]_[relationName]
```suggestion
                nullable: false, foreignKeyName: fk_enrollee_relation_participant_user, references: participant_user(id)} }
```